tests/ui/rfcs/rfc-0000-never_patterns/diverges.rs

```diff
@@ -27,3 +28,8 @@ fn never_match() -> u32 {
     }
     println!(); // Ensures this typechecks because of divergence.
 }
+
+// Divergence is not detected but at least nothing weirder happens.
```

Comment:
Please split this out into another test file so you can make it a `// known-bug:` + file an issue and link it w/ some tag (`F-never_patterns`?) so we don't forget this.